src/containers/Admin/EditTab/ZoomEditor/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { memo, useCallback, useState } from 'react'
+import ReactMapGL from 'react-map-gl'
+
+import { Slider } from '../../../../components'
+
+import './styles.scss'
+import { Label } from '@entur/typography'
+import { DEFAULT_ZOOM } from '../../../../constants'
+import { useSettingsContext } from '../../../../settings'
+
+function ZoomEditor(props: Props): JSX.Element {
+    const [settings, { setZoom }] = useSettingsContext()
+    const { zoom, onZoomUpdated } = props
+    const [viewport, setViewPort] = useState({
+        latitude: settings?.coordinates?.latitude,
+        longitude: settings?.coordinates?.longitude,
+        width: 'auto',
+        height: '40vh',
+        zoom: settings?.zoom ? settings?.zoom : DEFAULT_ZOOM,
+    })
+
+    const handleZoomUpdate = useCallback(
+        (event: React.ChangeEvent<HTMLInputElement>) => {
+            onZoomUpdated(Number(event.target.value))
+            setZoom(Number(event.target.value))
+            setViewPort({
+                latitude: settings?.coordinates?.latitude,
+                longitude: settings?.coordinates?.longitude,
+                width: 'auto',
+                height: '40vh',
+                zoom: settings?.zoom ? settings?.zoom : DEFAULT_ZOOM,
+            })
+        },
+        [onZoomUpdated, setZoom, settings],
+    )
+
+    return (
+        <div className="zoom-editor">
+            <Label>Hvor langt unna vil du at kartet skal være zoomet?</Label>
+            <Slider
+                handleChange={handleZoomUpdate}
+                distance={zoom}
+                min={13.5}
+                max={18}
+                step={0.1}
+            />
+            <p className="zoom-editor__text">
+                Kartet er zoomet in med nivå{' '}
+                <b>{Math.round((zoom - 12.5) * 10) / 10}</b>.
```

Comment:
Kvifor er 12.5 trekt frå her? Kor kjem det tallet frå?

Reply:
Rangen som er brukt er fra 13.5 til 18, da mye mindre i mapbox fort tar størrelsesorden by, land, kontinent. Trukket fra 12.5 for få 1-indeksert i visninga slik at det blir 1 til 5.5. Tenkte det kanskje ga litt mer mening for en potensiell bruker? Kan godt vise 13.5 til 18 også eller 0 til 4.5.